common/changes/leddie24-persona-imagearea-fix_2017-01-11-21-05.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Set explicit height for ms-Persona-ImageArea and ms-Persona-Image",
```

Comment:
Minor formatting change: "Persona: Set an explicit height for the image area and image."